docs/source/get_started/quick_start.md

```diff
@@ -0,0 +1,272 @@
+    main_language: python
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.17.3
+---
+
+(quick-start)=
+
+# Quick start
+
+In this tutorial we'll show the basics of running Quantum ESPRESSO calculations with AiiDA.
+Start by loading the ORM module and AiiDA profile:
+
+```python
+from aiida import orm, load_profile
+
+load_profile()
+```
+
+## Running your first `pw.x` calculation
+
+Let's first create a silicon structure - for example with [ASE](https://ase-lib.org/) - and store it in a `StructureData` node.
+
+```python
+from ase.build import bulk
+
+structure = orm.StructureData(ase=bulk('Si', 'fcc', 5.43))
+```
+
+Next, we'll load the `pw.x` code, which should have been set up in [basic installation instructions](#installation-setup-code).
+
+```python
+code = orm.load_code('pw@localhost')
+```
+
+The AiiDA Quantum ESPRESSO plugin comes with a set of tested protocols, which define default input parameters for different levels of precision.
+You can obtain pre-populated `builder` for most of the supported workflows calling the `get_builder_from_protocol` method:
+
+```python
+from aiida_quantumespresso.workflows.pw.base import PwBaseWorkChain
+
+builder = PwBaseWorkChain.get_builder_from_protocol(
+    code=code,
+    structure=structure,
+    protocol='fast'
+)
+builder
+```
+
+In short, the [Process Builder](https://aiida.readthedocs.io/projects/aiida-core/en/stable/topics/processes/usage.html#process-builder) is an AiiDA tool that allows you to set up the inputs for a process.
+Using the protocols, you get a fully populated one, that can be run immediately by the AiiDA engine:
+
+```python
+from aiida import engine
+
+results = engine.run(builder)
+```
+
+Because we selected the `fast` protocol, this calculation should finish quickly.
+The `results` contain the output nodes of the process.
+For example the calculated energy of the system in eV:
+
+```python
+results['output_parameters']['energy']
+```
+
+## Setting inputs
+
+In many cases you will want to adapt the inputs that are provided by the protocol.
+One way to do this is by providing an `overrides` dictionary:
+
+```python
+overrides = {
+    'pw': {
+        'parameters': {
+            'SYSTEM': {'nbnd': 10}
+        }
+    }
+}
+
+builder = PwBaseWorkChain.get_builder_from_protocol(
+    code=code,
+    structure=structure,
+    protocol='fast',
+    overrides=overrides
+)
+builder
+```
+
+:::{warning}
+
+Take care when setting inputs via the `overrides`!
+Currently, the `get_builder_from_protocol` method will simply ignore any `overrides` that are not used, for example when you make a typo, see: [https://github.com/aiidateam/aiida-quantumespresso/issues/1134](https://github.com/aiidateam/aiida-quantumespresso/issues/1134)
+
+:::
+
+Alternatively, you can also adapt the inputs on the `builder` afterwards:
+
+```python
+builder.pw.parameters['SYSTEM']['nbnd'] = 15
+builder
+
+```
+
+::::{note}
```

Comment:
As mentioned in the OP, I would move this information elsewhere. We can write a more dedicate tutorial/how to on "setting inputs".